src/components/NavBar/NavBar.tsx

```diff
@@ -1,26 +1,35 @@
 import { Link, useNavigate, useLocation, NavLink } from 'react-router-dom';
-import { useState, useEffect } from 'react';
+import { useEffect } from 'react';
 import styles from './NavBar.module.css';
-import Profile from '../Profile/Profile';
 import Notifs from '../Notifs/Notifs';
 import { getAvatarByID } from '../Avatar/Avatar';
 import {
   getUserDetailsAction,
   isloggedIn,
   loggedIn,
   user,
+  logout,
 } from '../../store/User/UserSlice';
 import { useAppDispatch, useAppSelector } from '../../store/hooks';
 import { AuthApi } from '@codecharacter-2023/client';
 import { apiConfig, ApiError } from '../../api/ApiConfig';
 import Toast from 'react-hot-toast';
+import DashboardOptions from '../DashboardOptions/DashboardOptions';
+import { cookieDomain } from '../../config/config.example';
+
+import sign_up from '../../assets/sign_up.png';
```

Comment:
Name your variables properly, if its an icon i need to know that from the name. Also use all the icons as svgs not pngs.